fastlane/Fastfile

```diff
@@ -0,0 +1,59 @@
+  # This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:android)
+
+platform :android do
+  desc "Runs all the tests"
+  lane :test do
+    gradle(task: "test")
+  end
+
+  lane :debug do
```

Comment:
Can we do the version incrementing here as well? We normally get QA debug build because we get better Logcat output from debug builds than we do from release builds.

Reply:
Agree. If we can basically copy paste the stuff from release into debug as well, that'd be awesome.